src/czbenchmarks/tasks/sequential.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from typing import List, Set
+
+from ..datasets import BaseDataset, DataType
+from ..metrics import metrics_registry
+from ..metrics.types import MetricResult, MetricType
+from ..models.types import ModelType
+from .base import BaseTask
+from .constants import RANDOM_SEED
+
+logger = logging.getLogger(__name__)
+
+
+class SequentialTask(BaseTask):
```

Comment:
Maybe a more descriptive name? `SequentialConsistencyTask` or `SequentialOrganization`? I'm not a comp bio, so maybe this is already sufficiently descriptive to the community.

Reply:
I agree that `SequentialOrganizationTask` would be more descriptive